wallet/wallet-cli-lib/src/config.rs

```diff
@@ -68,10 +69,15 @@ pub struct CliArgs {
     #[clap(long)]
     pub wallet_password: Option<String>,
 
-    /// Start staking after the start
-    #[clap(long)]
+    /// DEPRECATED: use start_staking_for_account instead!
+    /// Start staking for the DEFAULT account after starting the wallet
+    #[clap(long, requires("wallet_file"))]
     pub start_staking: bool,
 
+    /// Start staking for the specified account after starting the wallet
+    #[clap(long, requires("wallet_file"))]
+    pub start_staking_for_account: Option<U31>,
```

Comment:
It should be possible to start staking for multiple accounts. Not only one.

Reply:
changed to a vector, and modified a functional test to close the wallet and restart with staking for 2 accounts